Framework/DataObjects/inc/MantidDataObjects/MDEvent.h

```diff
@@ -203,19 +203,16 @@ template <size_t nd> class DLLExport MDEvent : public MDLeanEvent<nd> {
     totalErrSq = 0;
 
     size_t index(0);
-    typename std::vector<MDEvent<nd>>::const_iterator it = events.begin();
-    typename std::vector<MDEvent<nd>>::const_iterator it_end = events.end();
-    for (; it != it_end; ++it) {
-      const MDEvent<nd> &event = *it;
-      float signal = event.signal;
-      float errorSquared = event.errorSquared;
+    for (const MDEvent<nd> &evnt : events) {
```

Comment:
Keep old name (`event` instead of `evnt`) to make this change much smaller. Also, use `auto`?